packaging/snap/snap/snapcraft.yaml

```diff
@@ -176,26 +176,26 @@ parts:
       done
 
     build-packages:
-      - python3.8-venv
-      - python3.8
+      - python3.9-venv
+      - python3.9
     stage-packages:
       # Why so much packages ?
-      # Well because Python 3.8 is part of base image core20 so Snapcraft
+      # Well because Python 3.9 is part of base image core20 so Snapcraft
```

Comment:
Is the comment still relevant with 3.9 instead of 3.8

(I guess we may want to keep this hack even if 3.9 is not part of base image core20 just in case we will endup in the same issue when switching to core22...)